web-app/js/ala-image-viewer.js

```diff
@@ -3,11 +3,18 @@ var imgvwr = {};
 (function(lib) {
 
     var map_registry = {};
+    var imageServiceBaseUrl = "http://dev.ala.org.au:8080/ala-images";
 
     var base_options = {
-        imageServiceBaseUrl: "http://images.ala.org.au",
+        imageServiceBaseUrl:  "http://dev.ala.org.au:8080/ala-images",
         auxDataUrl: '',
-        initialZoom: 'auto'
+        auxDataTitle: 'View more information about this image',
+        initialZoom: 'auto',
+        addDownloadButton: true,
```

Comment:
I understand where you are coming from. One problem we have at the minute is that the image viewer has a different set of tools available in different places - and this has been pointed out as a bad thing by John La Salle.

With this in mind, i was thinking of a opt-out rather than an opt-in model. That way the developer has to make the decision to remove a tool rather than remembering to include all the tools that should be there.

I think for most apps in the ALA, all tools will be exposed. How does this sound ?